Tests/Kernel/TestKCOV.cpp

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (c) 2024, Space Meyer <git@the-space.agency>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <Kernel/API/kcov.h>
+#include <Kernel/Sections.h>
+#include <LibCore/System.h>
+#include <LibMain/Main.h>
+#include <LibTest/Macros.h>
+#include <LibTest/TestCase.h>
+#include <fcntl.h>
+#include <stdio.h>
+#include <sys/ioctl.h>
+#include <sys/mman.h>
+#include <unistd.h>
+
+// Fork. In the parent: wait for the child to exit and return true. In the child immediately return
+// false after the fork, which can then run the actual test. Only one thread per process can open
+// the KCOV device at the same time. LibTest runs the tests multithreaded.
+void fork_and_kill_parent();
```

Comment:
nitpick: no need for a declaration here, you can use an anonymous namespace or declare it `static`.

Reply:
marked it static.

Reply:
I can still see the declaration :thinking:

Reply:
Woops. I fixed the right thing in the wrong place. I thought you were commenting on crash_and_burn() here. Marked *both* as static now hehe.